specification/monitor/resource-manager/microsoft.insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -0,0 +1,961 @@
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "Action": {
+      "description": "An alert action.",
+      "properties": {
+        "actionGroupId": {
+          "type": "string",
+          "description": "the id of the action group to use."
+        },
+        "webhookProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "the dictionary of custom properties to include with the post operation. These data are appended to the webhook payload."
+          }
+        }
+      }
+    },
+    "MetricAlertProperties": {
+      "required": [
+        "enabled",
+        "description",
+        "severity",
+        "evaluationFrequency",
+        "windowSize",
+        "criteria"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "the description of the metric alert that will be included in the alert email."
+        },
+        "severity": {
+          "type": "integer",
+          "description": "Alert severity {0, 1, 2, 3, 4}"
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "the flag that indicates whether the metric alert is enabled."
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "the list of resource id's that this metric alert is scoped to."
+        },
+        "evaluationFrequency": {
+          "type": "string",
+          "format": "duration",
+          "description": "how often the metric alert is evaluated represented in ISO 8601 duration format."
+        },
+        "windowSize": {
+          "type": "string",
+          "format": "duration",
+          "description": "the period of time (in ISO 8601 duration format) that is used to monitor alert activity based on the threshold."
+        },
+        "criteria": {
+          "$ref": "#/definitions/MetricAlertCriteria",
+          "description": "criteria"
```

Comment:
`"description": "criteria"` is really vague and it causes a [validation error](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/356119128#L731) -- can you put in a better description?